SU-HSDO/suhumsci
# Arch 150 last menu dropdown off page

# READY FOR REVIEW

# Summary
- Fixes submenu issue of last two submenus sticking to the right of the viewport when not logged in, and lightly cleans up some spacing issues.

# Steps to Test
1. Examine the home page menu at various breakpoints.
2. Confirm that when you hover over one of the last two main menu items while at full screen, the submenu system remains under the main item, not way off to the right of the viewport.

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/ARCH-150)
